corehq/ex-submodules/casexml/apps/case/models.py

```diff
@@ -655,11 +655,12 @@ def _safe_replace_and_force_to_string(me, attr, val):
                                                               create_action)
         self.actions.append(create_action)
     
-    def update_from_case_update(self, case_update, xformdoc):
+    def update_from_case_update(self, case_update, xformdoc, other_forms=None):
```

Comment:
support passing more than just the primary form in memory - just had to push this down to get it to the call where it was needed.